activiti-cloud-service-common/activiti-cloud-service-common-config/src/test/java/org/activiti/cloud/common/error/attributes/ErrorAttributesMessageSanitizerTest.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2017-2020 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.activiti.cloud.common.error.attributes;
+
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class ErrorAttributesMessageSanitizerTest {
+
+    @ParameterizedTest
+    @ValueSource(strings = {
+        "An exception occurred @[java.lang.utils]",
+        "Could not serialize @[javax.validation]",
+        "Missing bean @[org.springframework]",
+        "Error at com.google.guava",
+        "Error at io.zipkin.brave",
+        "Error at org._int.obscurepackage"
+    })
+    void should_replaceMessageWithErrorNotDisclosed_when_matchesBlacklistedItems(String message) {
+        ErrorAttributesCustomizer sanitizer = getNewSanitizer();
```

Comment:
You can move `sanitizer` at the field level and initialize it there. JUnit will create a new instance of it for each test in the suite.

Reply:
Done in [56a57f4](https://github.com/Activiti/activiti-cloud/pull/905/commits/56a57f4c8040e95f9c1eb6a2a49979d0cdde2d16)